### ISSUE TITLE
Editor performance issues in large codebases

### ISSUE THREAD

**Issue by tonyxiao** [ecyrbe/zodios#416]
Using https://github.com/astahmer/openapi-zod-client and example schema https://api.merge.dev/api/accounting/v1/schema, we generated a zodios file that is about 5k lines long. At this length it seems that the autocomplete works only after a very significant lag, making it really difficult to use zodios. Does this happen to anyone else? Are there ways we can make the performance better?

**Comment by ecyrbe** [comment]
Hello,
I'll migrate this into Q&A because we are wrapping thé limites of typescript there.
So here are some thoughs:
- if you can, remove all the endpoints and schema you don't need on your code
- Split it into different files and different client zodios instance / like put a client for each resources (if you have isolated modules ts perf will go up)
